### PR TITLE
initial project setup

### PR DESCRIPTION
**Pull request by pradeepmaurya**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import HelloWorld from './components/HelloWorld.vue'
+</script>
+
+<template>
+  <div>
+    <a href="https://vitejs.dev" target="_blank">
+      <img src="/vite.svg" class="logo" alt="Vite logo" />
+    </a>
+    <a href="https://vuejs.org/" target="_blank">
+      <img src="./assets/vue.svg" class="logo vue" alt="Vue logo" />
```
Comment (ajaydewangan1100) [inline src/App.vue]: className changed

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
```
Comment (ajaydewangan1100) [inline .gitignore:7]: id not done

File: src/App.vue [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    </a>
+  </div>
+  <HelloWorld msg="Vite + Vue" />
+</template>
+
+<style scoped>
+.logo {
+  height: 6em;
```
Comment (ajaydewangan1100) [inline src/App.vue]: use pixel

**Review by ajaydewangan1100 — CHANGES_REQUESTED** [review]
change this comments

File: src/App.vue (hunk 3 above)
Comment (pradeepmaurya) [reply src/App.vue]: we are following em

**Review by ajaydewangan1100 — APPROVED** [review]

**Pull request merged**